src/perf_counters.cc

```diff
@@ -54,7 +54,7 @@ PerfCounters PerfCounters::Create(
     const int group_id = !is_first ? counter_ids[0] : -1;
     const auto& name = counter_names[i];
     if (name.empty()) {
-      GetErrorLogInstance() << "A counter name was the empty string\n";
+      VLOG(2) << "A counter name was the empty string\n";
```

Comment:
same here, as it results in the user not getting what they want.

Reply:
The behavior is same even if the user doesn't give any ```--benchmark_perf_counters``` command-line option. We need to differentiate between user-gave-empty-flag vs user-gave-no-flag.